WeakAurasOptions/ConditionOptions.lua

```diff
@@ -472,7 +598,7 @@ local function addControlsForChange(args, order, data, conditionVariable, condit
       type = "select",
       width = WeakAuras.normalWidth,
       values = WeakAuras.sound_types,
-      name = blueIfNoValue2(data, conditions[i].changes[j], "value", "sound", L["Differences"]),
+      name = blueIfNoValue2(data, conditions[i].changes[j], "value", "sound", L["Sound"], L["Sound"]),
```

Comment:
This seems out of scope for the ticket you’re trying to address, no? 

If you want to make changes like this, don’t forget to mention that you are making them.